packages/asset/contracts/AssetCreate.sol

```diff
@@ -40,31 +40,38 @@ contract AssetCreate is
     ICatalyst private catalystContract;
     AuthSuperValidator private authValidator;
 
-    // mapping of creator address to creator nonce, a nonce is incremented every time a creator mints a new token
+    /// @notice  mapping of creator address to creator nonce, a nonce is incremented every time a creator mints a new token
     mapping(address => uint16) public creatorNonces;
+    /// @notice mapping of minter address to signature nonce, a nonce is incremented every time asset minter consumes a signature generated by TSB
     mapping(address => uint16) public signatureNonces;
 
-    // A fee denoted in BPS that is charged when using the lazy minting feature, deducted from the creators payout
+    /// @notice  A fee denoted in BPS that is charged when using the lazy minting feature, deducted from the creators payout
```

Comment:
you put 2 spaces instead of one
```
@notice  A fee
```

Reply:
I see I did that in few places, nice catch!